src/components/Dashboard/OldDashbaord/GoalSetter/GoalForm.jsx

```diff
@@ -1,28 +1,48 @@
 import React, { useState } from "react";
 import { useDispatch } from "react-redux";
 import { createGoal } from "src/features/goals/goalSlice";
+import { toast } from "react-toastify";
+import styled from "styled-components";
+
+const GoalInput = styled.input`
+    width: 100%;
+    padding: 10px;
+    border: 1px solid ${(props) => (props.error ? "red" : "#e6e6e6")};
+    border-radius: 5px;
+    margin-bottom: 10px;
+    font-family: inherit;
+`;
 
 const GoalForm = () => {
     const [text, setText] = useState("");
+    const [error, setError] = useState(false);
 
     const dispatch = useDispatch();
 
     const onSubmit = (e) => {
         e.preventDefault();
+        if (text.trim() === "") {
+            toast("Cannot submit empty goal");
```

Comment:
```suggestion
            toast.warn("Cannot submit empty goal");
```
It will look better for this toast message